pkg/tfgen/test_data/installation-docs/invalid-example-expected.md

```diff
@@ -0,0 +1,9 @@
+This example is invalid and should not be translated for this test to pass
+
+## Example Usage
+
+
+
```

Comment:
I thought a goal of this PR was to remove empty `## Example Usage` sections like this.

Reply:
Yes! This test is only for the examples conversion. We would still have an empty Examples section after this process.